client/src/pages/Settings.jsx

```diff
@@ -61,145 +61,152 @@ const Searching = () => {
-                                <Icon
-                                    className='text-highlight'
-                                    icon="fluent:warning-20-filled"
-                                    height="16"
-                                    width="16"
-                                />
-                                <p className="dark:text-highlight text-red text-lg">Warning: You have unsaved settings</p>
-                            </div>
-                        </Animation.Bounce>
-                        <ButtonToolbar className="flex justify-end">
-                            <Animation.Fade in={hasUnsavedSettings}>
-                                <Button
-                                    appearance="primary"
-                                    onClick={cancelSettingsUpdate}
-                                    className='text-base font-normal bg-secondary font-normal w-24 h-9 rounded-md hover:bg-slate-700'
-                                >
-                                    Cancel
-                                </Button>
-                            </Animation.Fade>
-                             <Button
-                                 type="submit"
-                                 appearance="primary"
-                                 disabled={!hasUnsavedSettings}
-                                 className='text-base font-normal w-24 h-9 bg-blue-500 rounded-md hover:bg-blue-400'
-                             >
-                                Update
-                             </Button>
-                        </ButtonToolbar>
-                    </Form.Group>
-                </Form>
-            </div>
-        </div>
+        <div className="flex justify-center items-center flex-col min-w-[calc(100%-120px)] p-2 gap-5 bg-white dark:bg-primary min-h-full">
+			<div className="w-11/12 sm:w-[70%] md:w-[55%] lg:w-[35%] md:min-w-[300px] h-auto dark:bg-primary bg-light rounded-3xl px-6 py-6 md:px-14 md:py-20 border shadow-xl text-primary  dark:text-light ">
+				<Form onSubmit={handleSubmit} onChange={handleChange}>
+					<Form.Group className="flex justify-between" controlId="app-theme">
+						<div className="w-5/6">
+							<div className="flex">
+								<Icon
+									color={darkMode ? 'white' : 'gray'}
+									icon="ion:invert-mode"
+									height="18"
+									width="18"
+									className="mr-1.5"
+								/>
+								<Form.ControlLabel className="text-primary  dark:text-light  text-[12px] sm:text-xs md:text-base font-normal">
+									MODE
+								</Form.ControlLabel>
+							</div>
+							<Form.HelpText className="text-zinc-600 text-xs">
+								{darkMode ? 'Dark' : 'Light'} mode
+							</Form.HelpText>
+						</div>
+						<div className="flex items-center">
+							<Form.Control
+								className="flex justify-end items-center"
+								name="theme"
+								accepter={Toggle}
+								checked={settings.theme}
```

Comment:
@Shariar-Hasan when you resolve merge conflicts, you just don't remove the one that you don't want especially if there is a contraction between two codes, that's the lazy way, the good way is to try and understand what the other code does and see if it needs to be kept or removed.

```suggestion
								checked={settings.theme ?? true}
```